R/armax.inverse.sim.R

```diff
@@ -1,26 +1,71 @@
-## hydromad: Hydrological Modelling and Analysis of Data
```

Comment:
was this meant to disappear?

Reply:
The `@usage` problem was fixed by setting Lazydata: true in `DESCRIPTION`.

> Do we have a way to make sure that the Rd pages are kept in sync with the roxygen? Is this checked by CI?

R CMD check compiles all the documentation when it does the checks, so it should be fine. 

> was this meant to disappear?

That particular line is a comment rather than something that appears in the documentation - happy to put it back in!

Reply:
Yeah, we probably should put the copyright notice back in verbatim. Thanks!

Reply:
Done!